src/containers/Header/Header.tsx

```diff
@@ -9,6 +12,11 @@ export const Header = () => (
       <Logo />
       <Search />
       <Navigation />
+      <DefaultButton
+        kind="a"
+        leftIcon={<Icon as={BsTwitch} />}
+        link="https://www.igdb.com/oauth/twitch">
+        LOGIN WITH TWITCH</DefaultButton>
```

Comment:
time permitting, can look into i18next or a similar library so that copy is externalised

Reply:
Hmmm sorry but I don't understand yet what you're talking to me right now haha :P but will know soon ;)